docs/docs/commands/sync.md

```diff
@@ -32,6 +32,14 @@ notifications (security breaches, changes to service, etc).
 Upon success, you are also logged in :) Syncing should happen automatically from
 here!
 
+## Delete
+
+You can delete your sync account with
+
+```
+atuin unregister
```

Comment:
I've got some changes I'm making in this area, but I can make them once this is merged 😊